src/components/homePage/Contributors.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div id="contributors">
+    <h1>Created with love in 🇨🇦 by</h1>
```

Comment:
The "CA" isn't making the emoji show up for me. After some research, the Canadian flag doesn't even seem to be an emoji in the first place. It'd be great if you could insert an image in between the text :)

As well, this should be h2